--- SSDT-11.dsl~	2016-01-31 12:06:28.000000000 -0500
+++ SSDT-11.dsl	2016-01-31 12:07:04.000000000 -0500
@@ -42,15 +42,6 @@
     External (HDOS, MethodObj)    // Warning: Unresolved method, guessing 0 arguments
     External (HNOT, MethodObj)    // Warning: Unresolved method, guessing 1 arguments
 
-    /*
-     * External declarations that were imported from
-     * the reference file [../refs.txt]
-     */
-    External (_SB_.PCI0.LPCB.H_EC.ECRD, MethodObj)    // 1 Arguments
-    External (_SB_.PCI0.LPCB.H_EC.ECWT, MethodObj)    // 2 Arguments
-    External (_SB_.PCI0.PEG0.PEGP.SGPO, MethodObj)    // 2 Arguments
-    External (MDBG, MethodObj)    // 1 Arguments
-
     External (_SB_.BRTI, FieldUnitObj)
     External (_SB_.CSTE, FieldUnitObj)
     External (_SB_.LBTN, FieldUnitObj)
@@ -78,6 +69,15 @@
     External (S0ID, FieldUnitObj)
     External (SCIS, FieldUnitObj)
 
+    /*
+     * External declarations that were imported from
+     * the reference file [../refs.txt]
+     */
+    External (_SB_.PCI0.LPCB.H_EC.ECRD, MethodObj)    // 1 Arguments
+    External (_SB_.PCI0.LPCB.H_EC.ECWT, MethodObj)    // 2 Arguments
+    External (_SB_.PCI0.PEG0.PEGP.SGPO, MethodObj)    // 2 Arguments
+    External (MDBG, MethodObj)    // 1 Arguments
+
     OperationRegion (SANV, SystemMemory, 0xD5741E18, 0x018A)
     Field (SANV, AnyAcc, Lock, Preserve)
     {
