--- SSDT-11.dsl~	2016-01-24 11:33:29.000000000 -0500
+++ SSDT-11.dsl	2016-01-24 11:35:36.000000000 -0500
@@ -46,9 +46,6 @@
      * External declarations that were imported from
      * the reference file [../refs.txt]
      */
-    External (_SB_.PCI0.LPCB.H_EC.ECRD, MethodObj)    // 1 Arguments
-    External (_SB_.PCI0.LPCB.H_EC.ECWT, MethodObj)    // 2 Arguments
-    External (_SB_.PCI0.PEG0.PEGP.SGPO, MethodObj)    // 2 Arguments
 
     External (_SB_.BRTI, FieldUnitObj)
     External (_SB_.CSTE, FieldUnitObj)
@@ -76,7 +73,12 @@
     External (PNHM, FieldUnitObj)
     External (S0ID, FieldUnitObj)
     External (SCIS, FieldUnitObj)
+    External (_SB_.PCI0.LPCB.H_EC.ECRD, MethodObj)    // 1 Arguments
+    External (_SB_.PCI0.LPCB.H_EC.ECWT, MethodObj)    // 2 Arguments
+    External (_SB_.PCI0.PEG0.PEGP.SGPO, MethodObj)    // 2 Arguments
+
 
+    
     OperationRegion (SANV, SystemMemory, 0xD5741E18, 0x018A)
     Field (SANV, AnyAcc, Lock, Preserve)
     {
